### PR TITLE
Update taup introduction

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: source/taup/introduction.rst [hunk 1]
```diff
@@ -16,10 +16,12 @@ TauP 用于计算一维球状分层模型下地震震相的走时和路径。
 
 1. 计算震相走时、射线参数、反射点、穿透点、射线路径、走时曲线等
 2. 内置震相解析器，支持任意震相名
-3. 部分兼容 ttimes 震相名： ``ttp`` 表示常见 P 波震相；同理，还有 ``tts``、``ttp+``、``tts+``、``ttbasic``、``ttall``
-4. 自带多个地球地震波速度参考模型：iasp91、prem、ak135 等
+3. 部分兼容 ttimes 震相名： ``ttp`` 表示常见 P 波震相；同理，
+   还有 ``tts``、``ttp+``、``tts+``、``ttbasic``、``ttall``
```
Comment (core-man) [inline source/taup/introduction.rst:20]:
```suggestion
3. 部分兼容 ttimes 震相名： ``ttp`` 表示常见 P 波震相；
   同理，还有 ``tts``、``ttp+``、``tts+``、``ttbasic``、``ttall``
```

File: source/taup/introduction.rst [hunk 2]
```diff
@@ -30,19 +32,23 @@ TauP 目前还存在一些缺点：
 工具
 ----
 
-TauP 提供了多个工具:
+TauP 即提供了图形界面也提供了命令行工具。
```
Comment (core-man) [inline source/taup/introduction.rst:35]:
```suggestion
TauP 既提供了图形界面，也提供了命令行工具。
```

File: source/taup/introduction.rst [hunk 3]
```diff
@@ -30,19 +32,23 @@ TauP 目前还存在一些缺点：
...
 
 - 图形界面： ``taup``
 - 命令行工具：
 
   - :doc:`taup_time` ：计算震相的走时、射线参数、出射角、入射角等
   - :doc:`taup_curve` ：计算走时曲线
   - :doc:`taup_path` ：计算射线路径
```
Comment (core-man) [inline source/taup/introduction.rst:42]:
```suggestion
  - :doc:`taup_time`\ ：计算震相的走时、射线参数、出射角、入射角等
  - :doc:`taup_curve`\ ：计算走时曲线
  - :doc:`taup_path`\ ：计算射线路径
```

File: source/taup/introduction.rst [hunk 4]
```diff
@@ -30,19 +32,23 @@ TauP 目前还存在一些缺点：
...
-  - ``taup_pierce`` ：计算震相在波速不连续面和指定深度的反射点和穿透点
-  - ``taup_wavefront`` ：计算波前面
-  - ``taup_table`` ：计算走时表
+  - ``taup pierce`` ：计算震相在波速不连续面和指定深度的反射点和穿透点
+  - ``taup wavefront`` ：计算波前面
+  - ``taup table`` ：计算走时表
```
Comment (core-man) [inline source/taup/introduction.rst:45]:
```suggestion
  - ``taup pierce``\ ：计算震相在波速不连续面和指定深度的反射点和穿透点
  - ``taup wavefront``\ ：计算波前面
  - ``taup table``\ ：计算走时表
```

File: source/taup/introduction.rst [hunk 5]
```diff
@@ -30,19 +32,23 @@ TauP 目前还存在一些缺点：
...
   - :doc:`taup_setsac` ：将震相走时写入 SAC 头段变量中
-  - ``taup_create`` ：预计算速度模型
+  - ``taup create`` ：预计算速度模型
+  - ``taup phase`` ：通过语言描述震相的射线路径
+  - ``taup slowplot`` ：绘制速度模型的慢度-深度剖面
+  - ``taup velplot`` ：绘制速度模型的速度-深度剖面
+  - ``taup velmerge`` ：将部分速度模型与一个标准模型合并以构成一个新的速度模型
```
Comment (core-man) [inline source/taup/introduction.rst:51]:
```suggestion
  - ``taup create``\ ：预计算速度模型
  - ``taup phase``\ ：通过语言描述震相的射线路径
  - ``taup slowplot``\ ：绘制速度模型的慢度-深度剖面
  - ``taup velplot``\ ：绘制速度模型的速度-深度剖面
  - ``taup velmerge``\ ：将部分速度模型与一个标准模型合并以构成一个新的速度模型
```

File: source/taup/introduction.rst [hunk 6]
```diff
@@ -30,19 +32,23 @@ TauP 目前还存在一些缺点：
...
+  - :doc:`taup_time`\ ：计算震相的走时、射线参数、出射角、入射角等
+  - :doc:`taup_curve`\ ：计算走时曲线
+  - :doc:`taup_path`\ ：计算射线路径
+  - ``taup pierce``\ ：计算震相在波速不连续面和指定深度的反射点和穿透点
+  - ``taup wavefront``\ ：计算波前面
+  - ``taup table``\ ：计算走时表
   - :doc:`taup_setsac` ：将震相走时写入 SAC 头段变量中
-  - ``taup_create`` ：预计算速度模型
+  - ``taup create``\ ：预计算速度模型
+  - ``taup phase``\ ：通过语言描述震相的射线路径
```
Comment (core-man) [inline source/taup/introduction.rst:48]: It seems that `slowness = radius / vel` (s/rad).

**Review by core-man — APPROVED** [review]

File: source/taup/introduction.rst (hunk 6 above)
Comment (seisman) [reply source/taup/introduction.rst:48]: I don't know. Will explore it when writing the `taup slowplot` tutorial.

**Pull request merged**